src/jingle_sip/mod_jingle_sip.erl

```diff
@@ -284,6 +271,28 @@ translate_to_sip(<<"session-terminate">>, Jingle, Acc) ->
             {error, item_not_found}
     end.
 
+%% There is some mess with types hapenning in nksip_uac:invite/3
```

Comment:
didn't you fix it in your fork?

Reply:
nooo, there are still weird types in nksip.
 I've fixed one or two errors only.